application.example.yaml

```diff
@@ -214,7 +215,6 @@ trading:
         - NEO/USD
         - ONT/USD
         - DASH/USD
-      fee: 0.0010
```

Comment:
Was this intentional?

Reply:
I believe there was another ‘fee’ element in that same block so I removed the second one.

Reply:
In the `QuoineExchange` example there was only one `fee` and you removed it

Reply:
Ok I’ll put it back. 😬